x/uibc/uics20/memo_handler.go

```diff
@@ -85,9 +95,14 @@ func (mh *MemoHandler) onRecvPacketPrepare(
 }
 
 // runs messages encoded in the ICS20 memo.
+// Returns list of
 // NOTE: we fork the store and only commit if all messages pass. Otherwise the fork store
 // is discarded.
 func (mh MemoHandler) execute(ctx *sdk.Context) error {
+	if !mh.executeEnabled && (mh.isGMP || len(mh.msgs) == 0) {
```

Comment:
```suggestion
	if !mh.executeEnabled && (mh.isGMP || len(mh.msgs) != 0) {
```

becaue if msgs are 0 then it will not come here